client/app/(authenticated)/dashboard/page.tsx

```diff
@@ -24,73 +24,108 @@ type ContentItem = {
 
 export default function Page() {
   const { data: session } = useSession();
-  const role = session?.user?.role;
+  const role = session?.user?.app_role;
 
   const [contents, setContents] = useState<ContentItem[]>([]);
   useEffect(() => {
     if (typeof window !== "undefined") {
       const fetchData = async () => {
         let contentsModule;
         switch (role) {
-          case "admin":
-            contentsModule = await import("@/app/data/dashboard/admin.json");
+          case "cas_admin":
+            contentsModule = await import(
+              "@/app/data/dashboard/cas_admin.json"
+            );
+            break;
+          case "cas_analyst":
+            contentsModule = await import(
+              "@/app/data/dashboard/cas_analyst.json"
+            );
             break;
-          case "operator":
-            contentsModule = await import("@/app/data/dashboard/operator.json");
+          case "industry_user_admin":
+            contentsModule = await import(
+              "@/app/data/dashboard/industry_user_admin.json"
+            );
             break;
```

Comment:
I don't think I can ever get to the admin dashboard for industry_user since we don't have an app_role of "industry_user_admin". We only have the "industry_user" app_role & their permissions (admin or not admin) are defined by their relationship in the user_operator table.

I think this has effects on the route definitions as well (ie: `client/app/(authenticated)/bceidbusiness/industry_user_admin/dashboard/...`)

Would a solution be to only have an `industry_user` dasboard and routes, and then display things on their dashboard and make things readonly(non-admin)/editable(admin) based on their user_operator permissions within the page?

Reply:
I thought the "industry_user_admin" role could be defined based on something like [321](https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/321)

Let's discuss?